studio/pages/project/[ref]/auth/AuthFormSchema.tsx

```diff
@@ -0,0 +1,293 @@
+import { object, string } from 'yup'
```

Comment:
this file shouldn't be here in the `pages` folder

Reply:
nope. shouldnt be